src/arrayMethodSort.js

```diff
@@ -1,11 +1,17 @@
 'use strict';
 
-/**
- * Implement method Sort
- */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function(compareFunction = (a, b) =>
+    a.toString() > b.toString() ? 1 : -1) {
```

Comment:
Better will be to create a constant value for this default callback and then use it here. It will be easier to read